reproducible/mknodeimage.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
+
+alias python=usr/bin/python2.7
+WORKDIR="/workspace/nodejs_build"
+OUTPUTDIR="$WORKDIR/nodejs"
+mkdir -p "$OUTPUTDIR"
+cd "$WORKDIR"
+
+tar -xvf /node-v6.11.4.tar.gz
+cd node-v6.11.4
+./configure --prefix="$OUTPUTDIR"
+make -j4
+make install
+
+# pass -n to gzip to strip timestamps
+# strip the '.' with --transform that tar includes at the root to build nodejs
+GZIP="-n" tar --numeric-owner -czf /workspace/nodejs.tar.gz -C "$OUTPUTDIR" . --transform='s,^./,,' --mtime='1970-01-01'
+md5sum /workspace/nodejs.tar.gz
```

Comment:
You shouldn't need this md5sum, I just had it for debugging.

Reply:
done